src/app/pages/ArticlePage/ElectionBanner/index.test.tsx

```diff
@@ -25,6 +25,54 @@ describe('ElectionBanner', () => {
   describe.each(['canonical', 'amp'])('%s', platform => {
     const isAmp = platform === 'amp';
 
+    it.each(['live', 'test'])(
+      'should use the correct URL for the iframe when SIMORGH_APP_ENV is "%s"',
+      appEnv => {
+        if (appEnv === 'live') {
+          process.env.SIMORGH_INCLUDES_BASE_URL =
+            'https://www.bbc.com/ws/includes';
+          process.env.SIMORGH_INCLUDES_BASE_AMP_URL =
+            'https://news.files.bbci.co.uk';
+        }
+
+        if (appEnv === 'test') {
+          process.env.SIMORGH_INCLUDES_BASE_URL =
+            'https://www.test.bbc.com/ws/includes';
+          process.env.SIMORGH_INCLUDES_BASE_AMP_URL =
+            'https://news.test.files.bbci.co.uk';
+        }
+
+        process.env.SIMORGH_APP_ENV = appEnv;
+
+        const { getByTestId } = render(
+          <ElectionBanner aboutTags={mockAboutTags} />,
+          {
+            toggles: { electionBanner: { enabled: true } },
+            isAmp,
+          },
+        );
+
+        const wrappingEl = getByTestId(ELEMENT_ID);
+
+        const iframe = isAmp
+          ? wrappingEl.querySelector('amp-iframe')
+          : wrappingEl.querySelector('iframe');
```

Comment:
```suggestion
        const iframe = wrappingEl.querySelector('iframe, amp-iframe');
```

Reply:
This is neat, didn't realise it could this.